src/services/submit.ts

```diff
@@ -186,6 +186,27 @@ export default async function submit({
         return `${tenants.current.apiOrigin}/forms/${formSubmission.definition.id}/submissions/${data.submissionId}/pdf-document?accessToken=${data.pdfAccessToken}`
       }
     }
+
+    const downloadSubmissionPdfs = () => {
+      if (!data.pdfAccessToken) {
+        return undefined
+      }
+      if (
+        Array.isArray(
```

Comment:
and it's not an array?

Reply:
oh yeah I see, it's already defined in its own prop

Reply:
Updated.